features/svc-catalog_asb/asb.feature

```diff
@@ -142,6 +142,75 @@ Feature: Ansible-service-broker related scenarios
       | <%= cb.prefix %>-mariadb-apb    | <%= cb.prefix %>-mariadb-apb-credentials    |  dev    | <%= cb.prefix %>-mariadb-apb-parameters    | {"mysql_database":"admin","mysql_user":"admin","mariadb_version":"10.2","mysql_root_password":"test","mysql_password":"test"}         | mariadb    | # @case_id OCP-18241
       | <%= cb.prefix %>-mariadb-apb    | <%= cb.prefix %>-mariadb-apb-credentials    |  prod   | <%= cb.prefix %>-mariadb-apb-parameters    | {"mysql_database":"admin","mysql_user":"admin","mariadb_version":"10.2","mysql_root_password":"test","mysql_password":"test"}         | mariadb    | # @case_id OCP-18240
 
+  # @author chezhang@redhat.com
+  @admin
+  Scenario Outline: Multiple Plans support for DB APBs
+    # Get the registry name from the configmap
+    When I switch to cluster admin pseudo user
+    And I use the "openshift-ansible-service-broker" project
+    And I save the first service broker registry prefix to :prefix clipboard
+
+    # Swtich back to normal user and create first project
+    And I switch to the first user
+    Given I have a project
+
+    # Provision DB apb with dev plan
+    When I process and create:
+      | f | <%= ENV['BUSHSLICER_HOME'] %>/testdata/svc-catalog/serviceinstance-template.yaml                             |
+      | p | INSTANCE_NAME=<db_name>                                                                                      |
+      | p | CLASS_EXTERNAL_NAME=<db_name>                                                                                |
+      | p | PLAN_EXTERNAL_NAME=dev                                                                                       |
+      | p | SECRET_NAME=<db_secret_name>                                                                                 |
+      | p | INSTANCE_NAMESPACE=<%= project.name %>                                                                       |
```

Comment:
Extra white-spaces before the last `|`

Reply:
> Extra white-spaces before the last `|`

sorry, I can't figure it out. What is it should like?

Reply:
@cuipinghuo it looks like the extra empty space ```    When I process and create:
      | f | <%= ENV['BUSHSLICER_HOME'] %>/testdata/svc-catalog/serviceinstance-template.yaml                             |```

Reply:
@liangxia @pruan-rht thanks and finally know what is not right.